src/models/group.ts

```diff
@@ -0,0 +1,35 @@
+import { SvgInHtml } from "../types/svg";
+import Figure from "./figure";
+
+class Group extends Figure {
+    figures: Figure[];
+
+    constructor(svgEl: SvgInHtml) {
+        super(svgEl);
+        this.figures = [];
+        this.properties = {};
+    }
+
+    addFigure(figure: Figure) {
+        this.figures.push(figure);
+    }
+
+    removeFigure(figure: Figure) {
+        const index = this.figures.indexOf(figure);
+        if (index !== -1) {
+            this.figures.splice(index, 1);
+        }
+    }
+
+    refreshProperties() {
+        this.figures.forEach((figure) => {
+            Object.keys(this.properties).forEach((key) => {
```

Comment:
Properties should be attached to the group tag itself, not children; There is potential bug with the approach - create two circles move them across the board, them group. Expected: distance between them is same, actual: they overlap

Reply:
Yeah I saw what you mean.